Zend/zend_closures_arginfo.h

```diff
@@ -20,5 +20,5 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_class_Closure_call, 0, 0, 1)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_class_Closure_fromCallable, 0, 0, 1)
-	ZEND_ARG_INFO(0, callable)
+	ZEND_ARG_TYPE_INFO(0, callable, IS_CALLABLE, 0)
```

Comment:
It will make the the tests fail until https://github.com/php/php-src/pull/4883 is merged.

Reply:
That one turned out to be less simple than expected, probably best drop this type for now after all.

Reply:
OK!